src/ui/public/query_manager/query_manager.js

```diff
@@ -1,14 +1,15 @@
 import _ from 'lodash';
 
-export function queryManagerFactory(state) {
+export function queryManagerFactory(getState) {
 
   function getQuery() {
     return {
-      ...state.query
+      ...getState().query
     };
   }
 
-  function updateQuery(newQuery) {
+  function setQuery(newQuery) {
```

Comment:
While using this API in the tile map I kept trying to type setQuery instead of updateQuery. Seems like it's a better name.